azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/AddMojo.java

```diff
@@ -169,7 +169,7 @@ protected void doExecute() throws AzureExecutionException {
 
     //region Load templates
     protected BindingTemplate loadBindingTemplate(String type) {
-        try (final InputStream is = AddMojo.class.getResourceAsStream("/bindings.json")) {
+        try (final InputStream is = Constants.class.getResourceAsStream("/bindings.json")) {
```

Comment:
I think it could be better to provide a Util method in common library to get the Function template

Reply:
This PR is mainly for file move, let it be done in another PR